backend/open_webui/models/skills.py

```
@@ -0,0 +1,339 @@
+                db.add(result)
+                db.commit()
+                db.refresh(result)
+                AccessGrants.set_access_grants(
+                    "skill", result.id, form_data.access_grants, db=db
+                )
+                if result:
+                    return self._to_skill_model(result, db=db)
+                else:
+                    return None
+            except Exception as e:
+                log.exception(f"Error creating a new skill: {e}")
```

Ruff (0.15.0)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)